libminifi/test/SimpleSSLTestServer.h

```diff
@@ -50,11 +50,12 @@ class SimpleSSLTestServer  {
   };
 
  public:
-  SimpleSSLTestServer(const SSL_METHOD* method, int port, const std::filesystem::path& key_dir)
+  SimpleSSLTestServer(uint64_t disable_version, int port, const std::filesystem::path& key_dir)
```

Comment:
I think `tls_version_flags` would be a better parameter name than `disable_version`.

Reply:
Updated in 67eb58112fe948432498c39cae18aec78251fd22